src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3384,40 +3414,50 @@ private VariableState GetVariableState(Optional<LocalState> localState)
 
         private MethodSymbol InferMethodTypeArguments(BoundCall node, MethodSymbol method, ImmutableArray<BoundExpression> arguments)
         {
+            return InferMethodTypeArguments(node.BinderOpt, method, arguments, node.ArgumentRefKindsOpt, node.ArgsToParamsOpt, node.Expanded);
+        }
+
+        private MethodSymbol InferMethodTypeArguments(
+            Binder binder,
```

Comment:
>Binder binder, [](start = 12, length = 14)

do we still need to pass a binder here, given that the NullableWalker now has an instance field `_binder`?

Reply:
Perhaps not, although the existing caller passes the binder from the `BoundCall` node which may be distinct.

---
In reply to: [282702127](https://github.com/dotnet/roslyn/pull/35601#discussion_r282702127) [](ancestors = 282702127)

Reply:
If it is distinct, it would be good to comment. When should we use `_binder` as opposed to other binders we could get?

---
In reply to: [282748578](https://github.com/dotnet/roslyn/pull/35601#discussion_r282748578) [](ancestors = 282748578,282702127)